cmd/buildah/commit.go

```diff
@@ -20,6 +20,11 @@ var (
 			Usage: "don't compress layers",
 		},
 		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "Use `USERNAME[:PASSWORD]` for accessing the registry",
```

Comment:
Line 25 is now:

Usage: "use `username[:password]` for accessing the registry",

holler if that needs changing.